packages/compass-saved-aggregations-queries/src/stores/aggregations-queries-items.ts

```diff
@@ -125,7 +125,7 @@ const mapQueryToItem = (query: FavoriteQuery): Item => {
     lastModified: (query._dateModified ?? query._dateSaved).getTime(),
     database,
     collection,
-    type: 'query',
+    type: query.update ? 'updatemany' : 'query',
```

Comment:
I see this mapping, but no corresponding handling seems to be added anywhere in the compass-collection or crud, and so doesn't seem like trying to open an item from "my queries" actually does anything, at least it doesn't seem to work for me locally. Is it working for you?

Reply:
It is used in `packages/compass-saved-aggregations-queries/src/components/saved-item-card.tsx`, which despite it's name, it's the My Queries section.
<img width="920" alt="Screenshot 2023-11-13 at 16 40 32" src="https://github.com/mongodb-js/compass/assets/1822138/6315d7a9-13ac-4088-ae21-fe7a54ff2ffd">

Reply:
Yeah, I mean, it is passed to the event too, that's cool, but nothing is handling it in a way that would actually make it work, opening the saved update item doesn't open update modal currently